### ISSUE TITLE
single command applications with arguments

### ISSUE THREAD

**Issue by glensc** [mnapoli/silly#41]
while single command applications are possible with `setDefaultCommand`:

```php
$app->command('run', /* ... */);
$app->setDefaultCommand('run');
```

http://mnapoli.fr/silly/docs/command-definition.html

implemented (suggested) via: #30

it doesn't play well when you want to use arguments (not options), as that would require that you specify command name before first argument.

**Comment by glensc** [comment]
done some research,

seems it's [possible](https://github.com/eventum/eventum/pull/279#issuecomment-328251866) to solve this, but seems the secotnd argument to `setDefaultCommand`
  requires symfony/console 3.2

for older versions it's [possible](http://symfony.com/doc/3.1/components/console/single_command_tool.html) too, but a bit more complex.

neither of these is documented in http://mnapoli.fr/silly/docs/command-definition.html and doesn't mention problem with non-option arguments

**Comment by mnapoli** [comment]
Good point. I'd like to avoid the workaround for Symfony < 3.2 though. And upgrading to 3.2 is maybe a bit early?

**Comment by glensc** [comment]
i think documenting and proposing application to use 3.2 is sufficient. this library shouldn't require newer symfony (that is not base functionality). and pointing to documentation how to use in  older version.

yeah probably just putting my comment into docs:
https://github.com/mnapoli/silly/issues/41#issuecomment-328277595

**Comment by mnapoli** [comment]
:+1: then feel free to send a pull request to add that to the docs that would be perfect!
